docs/demo/scripts/kind/Makefile

```diff
@@ -122,9 +127,49 @@ kind-create: temp-dir kind kind-delete ## Create the Kind cluster
 	$(KIND) create cluster --config $(TEMP_DIR)/kind-config.yaml ; \
 	rm $(TEMP_DIR)/kind-config.yaml
 
+.PHONY: whereabouts
+whereabouts: temp-dir ## Install whereabouts IPAM CNI plugin
+	git clone https://github.com/k8snetworkplumbingwg/whereabouts $(TEMP_DIR)/whereabouts && \
+	kubectl apply \
+	-f $(TEMP_DIR)/whereabouts/doc/crds/daemonset-install.yaml \
+	-f $(TEMP_DIR)/whereabouts/doc/crds/whereabouts.cni.cncf.io_ippools.yaml \
+	-f $(TEMP_DIR)/whereabouts/doc/crds/whereabouts.cni.cncf.io_overlappingrangeipreservations.yaml && \
+	kubectl rollout status daemonset whereabouts -n kube-system --timeout 300s || \
+	{ echo >&2 "Error: Failed to install Whereabouts"; rm -fr $(TEMP_DIR)/whereabouts; exit 1; } ; \
+	rm -fr $(TEMP_DIR)/whereabouts
+
+.PHONY: multus
+multus: ## Install multus
+	kubectl apply -f https://raw.githubusercontent.com/k8snetworkplumbingwg/multus-cni/master/deployments/multus-daemonset.yml && \
+	kubectl rollout status daemonset kube-multus-ds -n kube-system --timeout 300s || { echo >&2 "Error: Failed to install Multus"; exit 1; }
+
+.PHONY: cni-plugins
+cni-plugins: temp-dir ## Install CNI plugins (add recovery option by failing cni install container if wget fails)
+	wget -qO $(TEMP_DIR)/cni-install.yml.j2 https://raw.githubusercontent.com/k8snetworkplumbingwg/multus-cni/master/e2e/templates/cni-install.yml.j2 && \
```

Comment:
why not applying (kubectl apply -f https://...) it directly with the URL? instead of downloading the file